pkg/api/types.go

```diff
@@ -2944,6 +2949,10 @@ const (
 	ResourceMemory ResourceName = "memory"
 	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
 	ResourceStorage ResourceName = "storage"
+	// Local Storage for overlay filesystem, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
+	ResourceStorageOverlay ResourceName = "storage.kubernetes.io/overlay"
```

Comment:
Clarify that the resource names are alpha and that they can change across releases. This is important because a new concept of `ResourceClasses` is being discussed where not having a domain name in the user resource name request will be useful.